### PR TITLE
SSVD-1823 - Z-wave Dimmer Switch (Generic)

### PR DESCRIPTION
**Pull request by juano2310**

### CODE REVIEW

File: devicetypes/smartthings/dimmer-switch.src/dimmer-switch.groovy [hunk 1]
```diff
@@ -224,4 +244,4 @@ def invertSwitch(invert=true) {
 	else {
 		zwave.configurationV1.configurationSet(configurationValue: [0], parameterNumber: 4, size: 1).format()
 	}
-}
\ No newline at end of file
+}
```
Comment (bflorian) [inline devicetypes/smartthings/dimmer-switch.src/dimmer-switch.groovy]: Indicator controls aren't doing a `sendHubCommand()`

**Comment by mckeed** [comment]
Looks like you missed the preferences section

**Comment by bflorian** [comment]
👍

**Comment by workingmonk** [comment]
if we know the manufacturer can't we make the deviceJoinName more specific?

Also, everyone working on sat? :)

**Comment by mckeed** [comment]
Yeah we wanted to get reasonable coverage for the SSVD ticket and we'll revisit the full list of fingerprints and join names. The main issue is we need to decide what to do with the GE vs Jasco branding.

**Pull request merged**